README.adoc

```diff
@@ -0,0 +1,142 @@
+= Hazelcast Viridian Maven Plugin
+:description: The Hazelcast Viridian Maven Plugin is a Java development tool for testing and deploying cluster-side modules on Serverless clusters.
+:page-plugin-version: 0.4-SNAPSHOT
+:page-serverless: true
+
+// This page is used as both the project README and the documentation reference at docs.hazelcast.com/cloud/maven-plugin.
+// To display content that is relevant for GitHub and internal developers only in the README, we use the env-github attribute, which is available only in the context of GitHub. For example, only internal developers need to know about the apiBaseUrl configuration element. This element is displayed only when env-github is true.
+
+ifndef::env-github[]
+The Hazelcast Viridian Maven Plugin is a Java development tool for testing and deploying xref:cluster-side-modules.adoc[cluster-side modules] on Serverless clusters.
+endif::[]
+
+ifdef::env-github[]
+image:https://maven-badges.herokuapp.com/maven-central/com.hazelcast.cloud/hazelcast-cloud-maven-plugin/badge.svg[link="https://maven-badges.herokuapp.com/maven-central/com.hazelcast.cloud/hazelcast-cloud-maven-plugin"]
+
+The Hazelcast Viridian Maven Plugin is a Java development tool for testing and deploying link:https://docs.hazelcast.com/cloud/cluster-side-modules[cluster-side modules] on Serverless clusters.
+endif::[]
+
+The current version of the plugin is {page-plugin-version}.
+
+== Before you Begin
+
+Make sure that you have the following:
+
+- link:https://maven.apache.org/install.html[Maven]
+- The `JAVA_HOME` environment variable set to the path of JDK 8, 9, or 11
+
+== Usage Example
+
+ifndef::env-github[]
+[source,xml,subs="attributes+"]
+----
+<pluginRepositories>
+    <pluginRepository>
+        <id>github</id>
+        <name>Hazelcast Maven Packages</name>
+        <url>https://maven.pkg.github.com/hazelcast/hazelcast-cloud-maven-plugin</url>
+    </pluginRepository>
+</pluginRepositories>
```

Comment:
`pluginRepositories` is no longer needed since plugin is released to the Maven Central repo.